### DISCUSSION TITLE
WebView2 Screen Share Position and Change Style

### DISCUSSION THREAD

**Discussion by lsyfg** [MicrosoftEdge/WebView2Feedback#4137] [Q&A]
Loading webview2 using WPF application。The shared screen window is fixed in the middle, hoping to change its position and change the window style。How to achieve？Can tab items be hidden in the window？

**Comment by victorhuangwq** [comment]
Hi, you won't be able to shift the shared screen window position and style if you want to use the native UI. 
If I am not wrong, @srwei is working on an API, that allows you to programmatically control it, but it also means that you have to create your own complete UI for it

The feature request for the screen share API is being tracked here https://github.com/MicrosoftEdge/WebView2Feedback/issues/2442

**Reply by lsyfg** [reply]
How to set hidden tab？
